customer/java/micronaut/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM oracle/graalvm-ce:1.0.0-rc7 as builder
+
+COPY . .
+
+RUN ./mvnw install:install-file -Dfile=/usr/java/latest/jre/lib/svm/builder/svm.jar -DgroupId=com.oracle.substratevm -DartifactId=svm -Dversion=GraalVM-1.0.0-rc7 -Dpackaging=jar
+RUN ./mvnw clean package
```

Comment:
It is required by Graal (v7) the version 8 and 9 will not require both but 8 and 9 does not work with micronaut. But notice that this is a multistage build so the result image does not contain all this information because how multistage build works in Maven.

Reply:
i did get multi stage build .. but did multistage build worked with  minishift docker daemon. i see it requires minimum docker version it never worked for me with minishift docker daemon